src/style.css

```
@@ -0,0 +1,126 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.product-img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.product-name {
+  margin: 40px 0 4px;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  letter-spacing: 0 em;
+  text-align: left;
+  color: #060b35;
+}
+
+.code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  letter-spacing: 0 em;
+  text-align: left;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.product-price,
+.product-response {
+  display: flex;
+  justify-content: space-between;
+}
+
+.product-response {
+  margin-bottom: 24px;
+}
+
+.response {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: flex-end;
+  color: #060b35;
+}
+
+.price {
+  width: 32px;
+  height: 18px;
+
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: #616070;
+}
+
+.value {
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 40px;
+  margin-top: 16px;
+
+  background: #00acdc;
+  border-radius: 5px;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  line-height: 16px;
+}
+
+.button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars :nth-child(-n+4) {
```

```suggestion
.stars :nth-child(-n+1),
.stars :nth-child(-n+2),
.stars :nth-child(-n+3),
.stars :nth-child(-n+4),
.stars :nth-child(-n+5) {
```

This should work, when i change amount of stars in html file